site2/website/release-notes.md

```diff
@@ -1,6 +1,107 @@
+* Fixed data is not deleted after expiration due to connected readers [#5621](https://github.com/apache/pulsar/pull/5621)
+* Fixed typo in BrokersBase file [#5596](https://github.com/apache/pulsar/pull/5596)
+* Fixed the go client docs missing in website [#5595](https://github.com/apache/pulsar/pull/5595)
+* Fixed schema def build error with protobuf schema [#5569](https://github.com/apache/pulsar/pull/5569)
+* Fixed docs about reset cursor [#5551](https://github.com/apache/pulsar/pull/5551)
+* Fixed repeated initialization of connectorsManager [#5545](https://github.com/apache/pulsar/pull/5545)
+* Fixed Functions unnecessarily restart during FunctionRuntimeManager init phase [#5527](https://github.com/apache/pulsar/pull/5527)
+* Fixed list non-persistent topics shows the persistent topics [#5502](https://github.com/apache/pulsar/pull/5502)
+* Return after triggering callback with empty result [#5500](https://github.com/apache/pulsar/pull/5500)
+* Fixed dispatcher skipping delivery of a batch during concurrent replays [#5499](https://github.com/apache/pulsar/pull/5499)
+* Fixed reader_listener option for Python API [#5487](https://github.com/apache/pulsar/pull/5487)
+* Fixed wrongly report "3600 messages have timed-out" [#5477](https://github.com/apache/pulsar/pull/5477)
+* Fixed broken custom auth-provider that uses authenticationData [#5462](https://github.com/apache/pulsar/pull/5462)
+* Fixed negative ack tracker constructor sequence [#5453](https://github.com/apache/pulsar/pull/5453)
+* Fixed StringSchema static initialization  [#5445](https://github.com/apache/pulsar/pull/5445)
+* Fixed message corruption on OOM for batch messages [#5443](https://github.com/apache/pulsar/pull/5443)
+* Fixed couple functions related integration tests [#5434](https://github.com/apache/pulsar/pull/5434)
+* Fixed bug that namespace policies does not take effect due to NPE [#5408](https://github.com/apache/pulsar/pull/5408)
+* Fixed race condition : Failed to read-more entries on dispatcher [#5391](https://github.com/apache/pulsar/pull/5391)
+* Fixed potential deadlock that can occur in addConsumer [#5371](https://github.com/apache/pulsar/pull/5371)
+* Fixed proxy to be able to re-send request body [#5361](https://github.com/apache/pulsar/pull/5361)
+* Fixed pulsar can't load the customized SerDe [#5357](https://github.com/apache/pulsar/pull/5357)
+* Fixed instability in Pulsar Function window integration test [#5337](https://github.com/apache/pulsar/pull/5337)
+* Fixed bk write failure part 2 [#5322](https://github.com/apache/pulsar/pull/5322)
+* Fixed bad_weak_ptr error when closing producer [#5315](https://github.com/apache/pulsar/pull/5315)
+* Fixed typo in prometheus metrics [#5299](https://github.com/apache/pulsar/pull/5299)
+* Fixed invalidate cache on zk-cache timeout [#5298](https://github.com/apache/pulsar/pull/5298)
+* Don't attempt to append on read-only cursor ledger [#5297](https://github.com/apache/pulsar/pull/5297)
```

Comment:
I think this PR is a enhancement,  Avoid persistent Unack Ranges while cursor recovering